### PR TITLE
Add -V as short for --version

### PR DESCRIPTION
**Pull request by hbina**
Related issue https://github.com/RedisLabs/memtier_benchmark/issues/139

TIL about Linux's [optarg](https://man7.org/linux/man-pages/man3/getopt.3.html).

### CODE REVIEW

**Comment by YaacovHazan** [comment]
@hbina why not just to fix the existing -v option?

**Comment by hbina** [comment]
Hmm well you are right, I could have just fixed the old 'v'. IDK which is preferred, but you can edit this commit if you prefer that.

**Comment by hbina** [comment]
I have applied your suggestion and fixed help string to also display short name for `--help. The diff is a lot smaller now.

File: memtier_benchmark.cpp [hunk 1]
```diff
@@ -874,15 +874,15 @@ void usage() {
             "  -x, --run-count=NUMBER         Number of full-test iterations to perform\n"
             "  -D, --debug                    Print debug output\n"
             "      --client-stats=FILE        Produce per-client stats file\n"
-            "      --out-file=FILE            Name of output file (default: stdout)\n"
+            "  -o  --out-file=FILE            Name of output file (default: stdout)\n"
```
Comment (yaacovhazan-Redislabs) [inline memtier_benchmark.cpp]: pls add ","

File: memtier_benchmark.cpp [hunk 2]
```diff
@@ -874,15 +874,15 @@ void usage() {
...
             "      --json-out-file=FILE       Name of JSON output file, if not set, will not print to json\n"
             "      --hdr-file-prefix=FILE     Prefix of HDR Latency Histogram output files, if not set, will not save latency histogram files\n"
             "      --show-config              Print detailed configuration before running\n"
             "      --hide-histogram           Don't print detailed latency histogram\n"
             "      --print-percentiles        Specify which percentiles info to print on the results table (by default prints percentiles: 50,99,99.9)\n"
             "      --cluster-mode             Run client in cluster mode\n"
-            "      --help                     Display this help\n"
-            "      --version                  Display version information\n"
+            "  -h  --help                     Display this help\n"
```
Comment (yaacovhazan-Redislabs) [inline memtier_benchmark.cpp]: pls add ","

**Review by yaacovhazan-Redislabs — CHANGES_REQUESTED** [review]

**Review by yaacovhazan-Redislabs — APPROVED** [review]

**Comment by yaacovhazan-Redislabs** [comment]
thank you @hbina for the PR... merged

**Pull request merged**